maths/fibonacci.py

```diff
@@ -230,6 +232,40 @@ def fib_binet(n: int) -> list[int]:
     return [round(phi**i / sqrt_5) for i in range(n + 1)]
 
 
+def fib_matrix(n: int) -> int:
+    # https://www.nayuki.io/page/fast-fibonacci-algorithms#:~:text=
+    # Summary:%20The%20two%20fast%20Fibonacci%20algorithms%20are%20matrix
```

Comment:
1. Could you make this into a docstring so that it serves as documentation as well?
2. This function needs unit tests, which should also go in the docstring.